### PR TITLE
Experimental "registry-index" command that uploads operations from API specs into a BigQuery table

### PR DESCRIPTION
**Pull request by timburks**
Let's consider this a very early experiment in exporting data to BigQuery. It seems likely that we'll want more, and that we'll want to either do this in the registry tool or in a related tool, but we'll start with `registry-bigquery` which could integrate nicely using the plugin mechanism.

This adds three indexing commands that index operations, servers, and information from the "info" block of OpenAPI specs.

### CODE REVIEW

File: cmd/registry-bigquery/index/info.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+					return err
+				}
+			}
+			return nil
+		},
+	}
+	cmd.Flags().StringVar(&filter, "filter", "", "Filter selected resources")
+	cmd.Flags().StringVar(&project, "project", "", "Project to use for BigQuery upload (defaults to registry project)")
+	cmd.Flags().StringVar(&dataset, "dataset", "registry", "BigQuery dataset name")
+	cmd.Flags().IntVar(&step, "step", 10000, "Step size to use when uploading records to BigQuery")
```
Comment (theganyo) [inline cmd/registry-bigquery/index/info.go:101]: Would "batch size" be a more standard and universally understood term?

File: cmd/registry-bigquery/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# registry-bigquery
+
+`registry-bigquery` is an experimental command-line tool that uses BigQuery to
```
Comment (theganyo) [inline cmd/registry-bigquery/README.md:3]: Maybe mention that this only recognizes and processes Open API specs?

**Review by theganyo — APPROVED** [review]

File: cmd/registry-bigquery/index/info.go (hunk 1 above)
Comment (timburks) [reply cmd/registry-bigquery/index/info.go:101]: Thanks, I was stuck trying to think of a single-word for this, do you think "batch" would work?

File: cmd/registry-bigquery/README.md (hunk 2 above)
Comment (timburks) [reply cmd/registry-bigquery/README.md:3]: Right, will do. Thanks!

File: cmd/registry-bigquery/index/info.go (hunk 1 above)
Comment (theganyo) [reply cmd/registry-bigquery/index/info.go:101]: Well, I don't think "--batch-size" is bad... but yes, I think "batch" would be clearer than "step".

Comment (timburks) [reply cmd/registry-bigquery/index/info.go:101]: ok, `--batch-size` SGTM

**Pull request merged**